views/idea.handlebars

```diff
@@ -0,0 +1,16 @@
+<br>
```

Comment:
Can this file get updated to use a bootstrap cards? Maybe one for the details and then one around the comments? See the idea partial or forms for examples, or check out bootstrap docs :)